assets/javascripts/modules/youtubePlayer.js

```diff
@@ -0,0 +1,83 @@
+require('jquery');
+var GOVUK = require('stageprompt');
+
+/**
+ * Embed a Youtube iframe in the page, replacing a div with the js-youtube-player class.
+ * Attributes required for player parameters:
+ *
+ *  id: required to render the iframe, value does not make a difference.
+ *  data-video-id: the id of the Youtube video to play.
+ *  data-ga-play-event: Google Analytics event to fire when video is played, should be in category:event:label format. omit if no GA event required.
+ *
+ * Usage example:
+ *
+ *   <div class='youtube-player-container'>
+ *      <div id='youtube-player' class='js-youtube-player' data-video-id='MlMnz0Omosk' data-ga-play-event='tax-return-help:Click:play'></div>
+ *   </div>
+ */
+
+module.exports = function() {
+
+  var YoutubePlayer = YoutubePlayer || {};
+  YoutubePlayer.started = false;
+  YoutubePlayer.player = {params: {}};
+
+  $('.js-youtube-player').each(function() {
+      YoutubePlayer.params = {
+        elementId: $(this).attr('id'),
+        videoId: $(this).data('video-id') || '',
+        gaEvent: $(this).data('ga-play-event') || ''
+      };
+
+      var tag = document.createElement('script');
+      tag.src = 'https://www.youtube.com/iframe_api';
+      var firstScriptTag = document.getElementsByTagName('script')[0];
+      firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+    }
+  );
+
```

Comment:
This is potentially running through a number of .js-youtube-player elements on the page but each time it will overwrite the previous version.
It will also add a new `https://www.youtube.com/iframe_api` for each .js-youtube-player element on the page.
For each player you need a unique identifier.